tests/exchange/test_binance.py

```diff
@@ -173,30 +173,32 @@ def test_stoploss_adjust_binance(mocker, default_conf, sl1, sl2, sl3, side):
 def test_get_max_leverage_binance(default_conf, mocker, pair, nominal_value, max_lev):
     exchange = get_patched_exchange(mocker, default_conf, id="binance")
     exchange._leverage_brackets = {
-        'BNB/BUSD': [[0.0, 0.025],
-                     [100000.0, 0.05],
-                     [500000.0, 0.1],
-                     [1000000.0, 0.15],
-                     [2000000.0, 0.25],
-                     [5000000.0, 0.5]],
-        'BNB/USDT': [[0.0, 0.0065],
-                     [10000.0, 0.01],
-                     [50000.0, 0.02],
-                     [250000.0, 0.05],
-                     [1000000.0, 0.1],
-                     [2000000.0, 0.125],
-                     [5000000.0, 0.15],
-                     [10000000.0, 0.25]],
-        'BTC/USDT': [[0.0, 0.004],
-                     [50000.0, 0.005],
-                     [250000.0, 0.01],
-                     [1000000.0, 0.025],
-                     [5000000.0, 0.05],
-                     [20000000.0, 0.1],
-                     [50000000.0, 0.125],
-                     [100000000.0, 0.15],
-                     [200000000.0, 0.25],
-                     [300000000.0, 0.5]],
+        'BNB/BUSD': [[0.0, 0.025, 0.0],
+                     [100000.0, 0.05, 2500.0],
+                     [500000.0, 0.1, 27500.0],
+                     [1000000.0, 0.15, 77499.99999999999],
+                     [2000000.0, 0.25, 277500.0],
+                     [5000000.0, 0.5, 1527500.0]],
+        'BNB/USDT': [[0.0, 0.0065, 0.0],
+                     [10000.0, 0.01, 35.00000000000001],
```

Comment:
If possible, i'd avoid this level of decimals in tests.
it's just a python float error - and i don't think removing the 15 decimals (in this case) will matter to the test outcome.

Reply:
I removed the ones that didn't break tests